### ISSUE TITLE
[feature] browsers with bigger default fonts should get bigger windows

### ISSUE THREAD

**Issue by masukomi** [masukomi/backup_brain_webextension#2]
The size of the window we open should be determined by the browser's default font size. That way low-vision folks who need larger fonts won't have half the text cut-off because the window is only reasonably sized for people with good vision and small fonts. 

This [stack overflow post](https://stackoverflow.com/questions/7444451/how-to-get-the-actual-rendered-font-when-its-not-defined-in-css/7444724#7444724) describes how to get the size of the default font.

```javascript
    function objectCssProperty( element, property ) {
        return window.getComputedStyle( element, null ).getPropertyValue( property );
    }
    function findDefaultFontSize(element){
       let cssValue = objectCssProperty(element, 'font-size')
       if (cssValue !== undefined  && cssValue.match(/\d+px/)){
          return Number(cssValue.replace(/px/, ''))
       } else {
          return 12;
      }
     }
```
usage
```javascript
objectCssProperty( object, 'font-size' ) // returns '16px' for instance
findDefaultFontSize(object) // returns 16 for instance
```

My thinking is that we have a default font size set in the preferences (integer only, & no ui to modify it) which is whatever FF & Chrome use by default. 
When you open the preferences window OR the menu window it checks the font size of one of the menu items. If that font size doesn't match what's in the preference we update the preference. This way if you open one from a right-click menu it'll (🤞) hopefully have an appropriate font size to multiply in order to get a good window size. 

So calculate these:  
```javascript

widthFontMultiplier = 67; //(800 / browser-default-font-size)

heightFontMultiplier = 29; // (350 / browser-default-font-size)
heightWithTagsFontMultiplier = 42; // (550 / browser-default-font-size)
```
then hardcode the results of ☝🏻 into the page 

so new window 
```javascript
const currentFontSize  = findDefaultFontSize(document.getElementById('font-size-tester'))
const defaultFontSize = Preferences.get('default_font_size')
if (currentFontSize != defaultFontSize){
  Preferences.set('default_font_size', currentFontSize);
}
const windowWidth = (defaultFontSize * widthFontMultiplier)
const windowHeight = (defaultFontSize * (show_tags ? heightWithTagsFontMultiplier : heightFontMultiplier))
const pin_window = await browser.windows.create({
            url: url,
            type: 'popup',
            width: windowWidth,
            height: windowHeight,
            incognito: bg_window.incognito
        })

```
